pkg/feature/templates/serverless/serving-install/knative-serving.tmpl

```diff
@@ -0,0 +1,23 @@
+apiVersion: operator.knative.dev/v1beta1
```

Comment:
another question to better understand things, what is the reasoning of using .tmpl instead yaml?

Reply:
Ah, I see it, ignore this comment :)